endoreg_db/serializers/__init__.py

```
@@ -6,5 +6,5 @@
 from .hardware import *
 from .label import *
 from .patient import *
-from .report_file import *
+from .documents import *
```

Ruff (0.8.2)

`from .documents import *` used; unable to detect undefined names

(F403)